docs/pywbemclicmdshelp.rst

```diff
@@ -858,13 +861,12 @@ The following defines the help output for the `pywbemcli connection list --help`
 
       List the WBEM connection definitions.
 
-      List the WBEM connection definitions in the connections file as a table.
-
-      In the table, the current connection is marked with an "*" after its name.
-
-      Examples:
+      This command displays all entries in the connections file and the current
+      connection if it is not in the connections file as a table using the
+      command line output_format to define the table format.
 
-        pywbemcli connection list
+      An "!" before the name indicates the selected connection. A '!' before the
+      name indicates that it is the current connection.
```

Comment:
Multiple issues:
1. This paragraph talks two times about '"!".
2. It is not clear what a "selected" connection is other than the new current connection. So I think there is no need to indicate a selected connection separately from the current connection.
3. You also introduced the concept of a default connectzion, but it is missing in this description.